fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/JDBCConstants.java

```diff
@@ -45,6 +45,7 @@
     public static final String CURRENT_RESOURCE_ID = "CURRENT_RESOURCE_ID";
     public static final String PARAMETER_NAME_ID = "PARAMETER_NAME_ID";
     public static final String IS_DELETED_NO = "IS_DELETED = 'N'";
+    public static final String IS_DELETED = "IS_DELETED";
```

Comment:
You could flip  flop 47/48 and build 47  and then 48. 
also there is an identical  string in ExpressionUtils... that should  be reconciled as a single constant.

Reply:
constant reconciliation to be done after our whole-system search change.